sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -590,6 +591,32 @@ def run_Flatten(self, transform_node, options):
           PropertyNames.ENCODING: step.encoding,
           PropertyNames.OUTPUT_NAME: PropertyNames.OUT}])
 
+  def apply_WriteToBigQuery(self, transform, pcoll, options):
```

Comment:
Please add a TODO to delete this method once new BQ sink is the default (and prob. also create a JIRA)

Reply:
Done.